docs/features/vpn.rst

```diff
@@ -0,0 +1,45 @@
+VPN
+===
+
+Gluon supports different options to establish vpn tunnels,
+which connect mesh clouds and provide internet access.
```

Comment:
not necessarily internet...

Reply:
> internetwork access

the problem with internet is … it's a shorthand for internetwork … but by now it's meaning has been superseded by the one global network.